packages/strapi-design-system/src/IconButton/IconButton.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { Tooltip } from '../Tooltip';
+import { BaseButton } from '../BaseButton';
+
+const IconButtonWrapper = styled(BaseButton)`
+  svg {
+    > g,
+    path {
+      fill: ${({ theme }) => theme.colors.neutral500};
+    }
+  }
+  &:hover {
+    svg {
+      > g,
+      path {
+        fill: ${({ theme }) => theme.colors.neutral600};
+      }
+    }
+  }
+  &:active {
+    svg {
+      > g,
+      path {
+        fill: ${({ theme }) => theme.colors.neutral400};
+      }
+    }
+  }
+`;
+
+export const IconButton = React.forwardRef(({ children, title, ...props }, ref) => {
+  return (
+    <Tooltip content={title}>
+      <IconButtonWrapper aria-label={title} {...props} ref={ref}>
```

Comment:
I think aria-label and the tooltip are redundant information. For now, tooltip is used to describe the element, but we may adjust it so that it becomes the label of the element. For instance, in this example, both the label and the description of the element provide the same information. Not sure this is what we exactly want 😊

Reply:
Not sure to clearly understand your comment sorry Marvin 🙈.

Reply:
When wanting to provide a meaning to element that are not written with text (e.g icons, button) or to provide more information on an element, we can use aria-label (and aria-labelledby) and aria-describedby.

In your case, both aria-labelledby and aria-describedby provide the exact same information ("Create", "Edit" etc...) I think it would be create to distiguish if we want the tooltip to be the main label (aria-label or aria-labelledby) or if it aims to provide addition information (aria-describedby).

Let me know if it's still not clear 😊